site/docs/doxygen/layout.xml

```diff
@@ -0,0 +1,241 @@
+<doxygenlayout version="1.0">
+  <!-- Generated by doxygen 1.8.17 -->
```

Comment:
Is this more than the default `layout.xml` from doxygen, or is there customization here? If it's the default I'm wondering about omitting it...

Reply:
It is very slightly different, as documented on lines 4-8. Would it be clearer if I removed the `Generated by doxygen` headers from any files I modified?

Reply:
Yeah, I think you should remove the generated-by lines to make it clear that there's customization.